static/js/settings_org.js

```diff
@@ -10,86 +10,6 @@ exports.reset = function () {
     meta.loaded = false;
 };
 
-var org_profile = {
-    name: {
-        type: 'text',
-    },
-    description: {
-        type: 'text',
-    },
-};
-
-var org_settings = {
-    msg_editing: {
-        allow_edit_history: {
-            type: 'bool',
-        },
-        allow_community_topic_editing: {
-            type: 'bool',
-        },
-    },
-    other_settings: {
-        inline_image_preview: {
-            type: 'bool',
-        },
-        inline_url_embed_preview: {
-            type: 'bool',
-        },
-        mandatory_topics: {
-            type: 'bool',
-        },
-        video_chat_provider: {
-            type: 'text',
-        },
-        google_hangouts_domain: {
-            type: 'text',
-        },
-        zoom_user_id: {
-            type: 'text',
-        },
-        zoom_api_key: {
-            type: 'text',
-        },
-        zoom_api_secret: {
-            type: 'text',
-        },
-        message_content_allowed_in_email_notifications: {
-            type: 'bool',
-        },
-    },
-    user_defaults: {
-        default_language: {
-            type: 'text',
-        },
-        default_twenty_four_hour_time: {
-            type: 'bool',
-        },
-    },
-    notifications: {
-        send_welcome_emails: {
-            type: 'bool',
-        },
-    },
-};
-
-var org_permissions = {
-    user_identity: {
-        name_changes_disabled: {
-            type: 'bool',
-        },
-        email_changes_disabled: {
-            type: 'bool',
-        },
-    },
-    other_permissions: {
-        bot_creation_policy: {
-            type: 'integer',
-        },
-        email_address_visibility: {
-            type: 'integer',
-        },
```

Comment:
Reading this, I'm curious -- should we be setting `data-setting-widget-type="integer"` for all the `select` elements?

Reply:
(Might be a good follow-up project, as opposed to something to do in this PR -- try to make something like `settings_checkbox.handlebars` for select widgets)

Reply:
Yeah, I guess maybe we can do make a generic widget for our select elements, I'll analyze the patterns and will be following up regarding its implementation(And if we do that then I guess we can set `data-setting-widget-type="integer"` for all the `select` elements.)